packages/website/package.json

```diff
@@ -24,8 +24,8 @@
         "vite": "^5.3.0"
     },
     "dependencies": {
-        "@svelte-add/adders": "workspace:^",
-        "@svelte-add/config": "workspace:^",
-        "@svelte-add/core": "workspace:^"
+        "@svelte-add/adders": "workspace:*",
```

Comment:
Should these remain as `^`? I'm not as familiar with the website package

Reply:
For the website, it doesn't really matter since they'll always be using the latest in `main`, so either method is fine

Reply:
Giving it some more thought, I think a good litmus test for this would be to ask: "should these be bundled?". If so, then we should leave it with a `workspace:*`. If not, then we should either consider using `workspace:^`, or perhaps making the dep a `peerDependency` _and_ use `workspace:^` (I'm thinking perhaps `testing-library` should be a peer-dep instead).

Reply:
My worry was that changesets might think versions need to be bumped more often when using `*`. I don't know if that's really something that will be an issue, but thought perhaps `^` could be safer from that respect. Otherwise I agree they should be equivalent when used in these packages that aren't published

Reply:
AFAIK, changesets isn't really concerned with how specific versions of packages relate to each other. All it cares about is who is a dependent of who, so the range specifier really doesn't matter in that regard

Reply:
Yeah, changesets shouldn't care about this change.